<launch>
    <node name="can_transceiver" pkg="roscan" type="can_transceiver" respawn="true" output="screen">
        <param name="can_log" type="bool" value="false" />
        <param name="buffer_log" type="bool" value="false" />
    </node>
</launch>
